### PR TITLE
Return a different error message if --provisioning flag is not set

### PR DESCRIPTION
**Pull request by mchavez**
- Right now if you try to provision something without the --provisioning flag, we error with "resource type does not have provisioner configured".  This error message should be something like "provisioning is not enabled. try running with --provisioning".

Fixes https://github.com/ConductorOne/baton-sdk/issues/114.

### CODE REVIEW

File: pkg/connectorbuilder/connectorbuilder.go [hunk 1]
```diff
@@ -317,8 +317,8 @@ func (b *builderImpl) Grant(ctx context.Context, request *v2.GrantManagerService
 		return &v2.GrantManagerServiceGrantResponse{Annotations: annos, Grants: grants}, nil
 	}
 
-	l.Error("error: resource type does not have provisioner configured", zap.String("resource_type", rt))
-	return nil, fmt.Errorf("error: resource type does not have provisioner configured")
+	l.Error("error: provisioning is not enabled. try running with --provisioning", zap.String("resource_type", rt))
```
[the remark this answers is not in the document]
Comment (mchavez) [reply pkg/connectorbuilder/connectorbuilder.go:318]: I added some changes. Now It shows "error: provisioning is not enabled. try running with --provisioning" in both cases, including service mode.

**Review by mgaeta — APPROVED** [review]

**Comment by mgaeta** [comment]
@mchavez You'll need to rebase this before you can merge.

**Comment by mgaeta** [comment]
@mchavez Is it ok if I close this PR?

**Comment by mchavez** [comment]
> @mchavez Is it ok if I close this PR?

@mgaeta  Not quite sure, because this PR is old. However, It seems like those changes are already implemented.

**Comment by mchavez** [comment]
Duplicated PR

**Pull request closed without merging**